dev/TeachingTip/TeachingTip.cpp

```diff
@@ -918,7 +922,12 @@ void TeachingTip::OnShouldConstrainToRootBoundsChanged()
     // ShouldConstrainToRootBounds is a property that can only be set on a popup before it is opened.
     // If we have opened the tip's popup and then this property changes we will need to discard the old popup
     // and replace it with a new popup.  This variable indicates this state.
-    m_createNewPopupOnOpen = true;
+
+    //The underlying popup api is only available on 19h1 plus, if we aren't on that no opt.
+    if (SharedHelpers::Is19H1OrHigher())
```

Comment:
Why does it matter if we recreate the popup in this case? Feel like the logic is simpler / easier to test if we just let it be recreated.

Reply:
It seemed wasteful to create a new popup every time the tip was opened.

Reply:
If the check was consistent with the other one (checking for the presence of the interface that contains ShouldConstrainToRootBounds) then I'd feel better about it. But still it seems like an extra check that just complicates the code. Sure it's wasteful but we're doing it on 19h1+ so why not just always do it? Consistency trumps minor perf here for me.